cmd/controller/app/options/options.go

```diff
@@ -134,6 +137,9 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 		"When this flag is enabled, the following sources for credentials are also used: "+
 		"AWS - All sources the Go SDK defaults to, notably including any EC2 IAM roles available via instance metadata.")
 
+	fs.StringVar(&s.DefaultACMEAnnotation, "default-acme-annotation", defaultACMEAnnotation, ""+
```

Comment:
Can we call this argument something more like `auto-certificate-annotation`? and perhaps make it a list too? (where if any of them match, we create a certificate for the ingress)

This annotation is technically not specific to acme, despite it currently being `tls-acme` 🙃 (users can configure any type of issuer as the default issuer to be used in ingress-shim, even if it is not an ACME issuer).

We can then in future invent a new annotation (e.g. `certmanager.k8s.io/auto-create-certificate: "true"`) and gradually deprecate the old `kubernetes.io/tls-acme` one.

Reply:
Also corresponding changes to var names throughout the codebase would be great 😄